src/core/qgsmapsettings.h

```diff
@@ -864,6 +865,22 @@ class CORE_EXPORT QgsMapSettings : public QgsTemporalRangeObject
      */
     void setCurrentFrame( long long frame );
 
+    /**
+     * Returns the shading renderer used to render shading on the entire map
+     *
+     * \see setElevationShadingRenderer()
+     * \since QGIS 3.30
+     */
+    const QgsElevationShadingRenderer &elevationShadingRenderer() const;
+
+    /**
+     * Sets the shading renderer used to render shading on the entire map
+     *
+     * \see elevationShadingRenderer()
+     * \since QGIS 3.30
+     */
+    void setElevationShadingRenderer( const QgsElevationShadingRenderer &elevationShadingRenderer );
```

Comment:
```suggestion
     * Sets the shading \a renderer used to render shading on the entire map
     *
     * \see elevationShadingRenderer()
     * \since QGIS 3.30
     */
    void setElevationShadingRenderer( const QgsElevationShadingRenderer &renderer );
```